### PR TITLE
dev: validate test configurations

### PR DESCRIPTION
**Pull request by ldez**
Another step related to Schema JSON validation: checking the configuration used by tests.

The PR fixes the configuration but also fixes the JSON schema.

one commit, one change.

### CODE REVIEW

File: test/configuration_file_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	"strings"
+	"testing"
+
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+func Test_validateTestConfigurationFiles(t *testing.T) {
+	err := validateTestConfigurationFiles("../jsonschema/golangci.next.jsonschema.json", ".")
```
Comment (Antonboom) [inline test/configuration_file_test.go:18]: does it depend on directory where `go test` was executed?

I remember 
```go
 _, filename, _, _ := runtime.Caller(0)
```
pattern for tests using related paths

File: test/configuration_file_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	require.NoError(t, err)
+}
+
+func validateTestConfigurationFiles(schemaPath, targetDir string) error {
+	schema, err := loadSchema(filepath.FromSlash(schemaPath))
+	if err != nil {
+		return err
```
Comment (Antonboom) [inline test/configuration_file_test.go:25]: no wrapping everywhere :(

**Review by Antonboom — APPROVED** [review]
👍

File: test/configuration_file_test.go (hunk 1 above)
Comment (ldez) [reply test/configuration_file_test.go:18]: It depends on the test file position.

**Pull request merged**